dali/pipeline/operators/decoder/nvjpeg_allocator.h

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Could you split these 2 allocators to separate header files?

Reply:
Sure
Will do during the ` dali/pipeline/operators/decoder/` files refactoring I plan to do once everything is fixed here